core/src/main/java/com/linecorp/armeria/common/util/SystemInfo.java

```diff
@@ -154,7 +154,7 @@ public static long currentTimeMicros() {
         return JavaVersionSpecific.get().currentTimeMicros();
     }
 
-    private static boolean isLinux() {
+    public static boolean isLinux() {
```

Comment:
Please add javadoc for `public` method.

Reply:
I wrote an annotation. 😃 
But is this correct ❔